### PR TITLE
Add method to set max cache size in ol.style.IconImageCache

### PR DESCRIPTION
**Pull request by bobobobo**
This adds a method to set max cache size as requested in #7233

### CODE REVIEW

**Comment by fredj** [comment]
travis is stuck; @bobobobo can you try to push force you branch to restart it?

**Comment by bobobobo** [comment]
@fredj Do you want me to rebase and force push?

**Comment by fredj** [comment]
> @fredj Do you want me to rebase and force push?

What I usually do to push force is:
```
git commit --amend
(save)
git push --force <remote> <branch>
```

File: src/ol/style/iconimagecache.js [hunk 1]
```diff
@@ -91,3 +90,13 @@ ol.style.IconImageCache.prototype.set = function(src, crossOrigin, color, iconIm
   this.cache_[key] = iconImage;
   ++this.cacheSize_;
 };
+
+
+/**
+ * Set cache max size.
+ * @param {number} maxCacheSize Cache max size.
+ */
+ol.style.IconImageCache.prototype.setMaxCacheSize = function(maxCacheSize) {
```
Comment (ahocevar) [inline src/ol/style/iconimagecache.js]: What about naming this nicer (`setSize`) and adding an `@api` annotation and some more verbose documentation?

**Comment by ahocevar** [comment]
I took the liberty to make the changes I suggested.

**Pull request merged**